lib/util/getClientVersion.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+const { platform } = require('os')
+
+const packageVersion = require('../../package.json').version
+
+const getClientVersion = () => {
+  const { version } = process
+  return `EthereumJS/${packageVersion}/${platform()}/node${version.substring(1)}`
+}
+
+module.exports = getClientVersion
```

Comment:
Let's make this file a bit more generic and not call it by the function name (if there is not an explicit intention behind it). Would suggest `info.js`, but also open for other/better alternatives.

Can you also add an export to the `util/index.js` file please?

Reply:
Right. I'll do it :+1: